cirq-google/cirq_google/op_deserializer.py

```diff
@@ -158,3 +208,91 @@ def _args_from_proto(
             if value is not None:
                 return_args[arg.constructor_arg_name] = value
         return return_args
+
+
+class CircuitOpDeserializer(OpDeserializer):
+    """Describes how to serialize CircuitOperations."""
+
+    @property
+    def serialized_id(self):
+        return 'circuit'
+
+    def from_proto(
+        self,
+        proto: v2.program_pb2.CircuitOperation,
+        *,
+        arg_function_language: str = '',
+        constants: List[v2.program_pb2.Constant] = None,
+        deserialized_constants: List[Any] = None,
+    ) -> 'cirq.CircuitOperation':
+        """Turns a cirq.google.api.v2.CircuitOperation proto into a CircuitOperation."""
```

Comment:
Add an Args section or at least point out that constants should have been previously parsed.

Reply:
Copied the docstring down from the superclass method and added changes as necessary. Same for the `from_proto` method of the `GateOpDeserializer`.